Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DscData.kt

```diff
@@ -1,7 +1,9 @@
 package de.rki.coronawarnapp.covidcertificate.signature.core
 
 import okio.ByteString
+import org.joda.time.Instant
 
 data class DscData(
-    val dscList: List<Pair<ByteString, ByteString>>
+    val dscList: List<Pair<ByteString, ByteString>>,
+    val updatedAt: Instant,
```

Comment:
@jurajkusnier You are doing the repo right? Could you provide such an `updatedAt` field?

Reply:
sure, I can add it
btw I will change `dscList` type to `List<DscListItem>` instead of `Pair<>`

Reply:
what should be in `updatedAt` when we load data from `DefaultDscSource` ?

Reply:
`Instant.EPOCH`